packages/builder/src/builder.js

```diff
@@ -420,7 +420,7 @@ export default class Builder {
       )
       // Only overwrite routes when something is returned for backwards compatibility
       if (extendedRoutes !== undefined) {
-        templateVars.router.routes = extendedRoutes
+        templateVars.router.routes = await extendedRoutes
```

Comment:
I think would be better to put await [here](https://github.com/nuxt/nuxt.js/blob/e10b9a72327d5c85ca93f64da490d666cd55f92b/packages/builder/src/builder.js#L417) because result of the promise can be undefined too

Reply:
Moved